stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/images/CoilStreamImageLoader.kt

```diff
@@ -69,7 +69,13 @@ internal object CoilStreamImageLoader : StreamImageLoader {
         val context = target.context
         val disposable = target.load(data, context.streamImageLoader) {
             headers(imageHeadersProvider.getImageRequestHeaders().toHeaders())
-            placeholderResId?.let(::placeholder)
```

Comment:
We just had a placeholder here, but that meant that if a load fails we would in certain situations display nothing so I added `fallback` (when the data is null) and `error` (when it errors out) values as well.

This seems to cover cancellation too since calling dispose early would make the placeholder appear.

Let me know if you think we might want to incorporate separate assets into `MessageListItemStyle` for fallback and error.